bucket/lunacy.json

```
@@ -22,7 +22,7 @@
     ],
     "checkver": {
         "url": "https://docs.icons8.com/release-notes/",
-        "regex": "LunacySetup_([\\d.]+)\\.exe\">here"
+        "regex": "LunacySetup_([\\d.]+)\\.exe\">Installer"
```

```suggestion
        "regex": "LunacySetup_([\\d.]+)\\.exe
```

What if using this regex?